### PR TITLE
Issue #1946: Add check for customer user password

### PR DESCRIPTION
**Pull request by stefanhaerter**
Closing #1946

### CODE REVIEW

**Review by svenoe — CHANGES_REQUESTED** [review]
This would mean that passwords cannot be deleted.

**Comment by stefanhaerter** [comment]
Ah, I forgot this case. if (defined ...) should fix this, right?

File: Kernel/Modules/AdminCustomerUser.pm [hunk 1]
```diff
@@ -371,6 +371,7 @@ sub Run {
                 );
             }
 
+            $CurrentUserData{UserPassword} //= '';
             if ( $CurrentUserData{UserPassword} ne $GetParam{UserPassword} ) {
```
Comment (svenoe) [inline Kernel/Modules/AdminCustomerUser.pm]: Please don't change the variable, instead use:
if ( ( undef // "123" ) eq "123" ) {print "klappt\n"}

**Comment by stefanhaerter** [comment]
Closing in favor of #2554

**Pull request closed without merging**